### PR TITLE
fix: UI refresh problem after mount local smb address

### PR DESCRIPTION
**Pull request by shzhuangpub**
If fetched smb network address and local address are pointed to the save share folder, just fetch the network address.

Log: 修复挂载本机smb地址后，在smb目录创建文件，无法显示
Bug: https://pms.uniontech.com/bug-view-145465.html

### CODE REVIEW

File: src/dde-file-manager-lib/gvfs/networkmanager.cpp [hunk 1]
```diff
@@ -400,8 +400,23 @@ void NetworkManager::fetchNetworks(const DFMUrlBaseEvent &event)
...
+        //因此应排除这种情况，不应赋值doChangeCurrentUrl = true去再次执行DFMChangeCurrentUrlEvent事件，
+        //否则会引起函数QModelIndex DFileSystemModel::setRootUrl(const DUrl &fileUrl)进入时，上一个d->jobController未结束，
+        //调用setParent函数时遇父子线程不一致，槽函数_q_onFileCreated()与watcher的连接失效，导致响应不到subfileCreated信号，
+        //界面不能及时刷新（fix bug:#145465）。
+        //例如，当出现bug #145465时，情况如下，temPath = shareName：
+        //DUrl(path) = DUrl("smb://x.x.x.x/test33")
+        //p1->getMountPointUrl() =  DUrl("file:///run/user/1000/gvfs/smb-share:server=x.x.x.x,share=test33")
+        QString temPath = DUrl(path).path();
+        temPath = temPath.mid(1);
+        QString shareName = FileUtils::smbAttribute(p1->getMountPointUrl().path(),FileUtils::SmbAttribute::kShareName);
```
Comment (Johnson-zs) [inline src/dde-file-manager-lib/gvfs/networkmanager.cpp:415]: use reference

**Comment by shzhuangpub** [comment]
该pr关闭，由下面这个替代：
https://github.com/linuxdeepin/dde-file-manager/pull/634

**Pull request closed without merging**